doc/source/index.rst

```diff
@@ -86,4 +87,13 @@ installed when using the same pip commands as above.
    :hidden:
    :maxdepth: 2
 
-   tutorials
\ No newline at end of file
+   tutorials
+
+**Mac OSX**
+
+Requires a brew install of the llvm-clang compiler,
+before running pip install run:
+
+.. code-block:: bash
+
+    brew install llvm
```

Comment:
Could this bit go before the `.. toctree::` stuff? I just like to have the fancy .rst things at the bottom of the page so they don't get lost in the main body of the documentation.

Reply:
Absolutely, my bad I had missed that the toctree wasn't part of the linux bits. Corrected now!